interpreters/wamr/Kconfig

```diff
@@ -47,6 +47,16 @@ config INTERPRETERS_WAMR_AOT_QUICK_ENTRY
 		signature.
 		Enable this option will increase the size of runtime, ~8KB.
 
+config INTERPRETERS_WAMR_AOT_WORD_ALIGN_READ
+	bool "Make WAMR AOT loader use word-aligned read"
+	default y
+	depends on INTERPRETERS_WAMR_AOT && ARCH_CHIP_ESP32S3
```

Comment:
why check ARCH_CHIP_ESP32S3 in common code?

Reply:
because these things in wamr are specific to esp32s3 at least for now.

Reply:
please remove esp32s3 specific code from wamr and remove ARCH_CHIP_ESP32S3 from here.

Reply:
i'm not happy with the current structure either.
but i want to make it work first.

Reply:
can we remove ARCH_CHIP_ESP32S3 from patch? espressif is working on to make the change in wamr more general for other arch/chip too.

Reply:
ok. done.